torchvision/prototype/transforms/functional/_utils.py

```diff
@@ -9,10 +8,10 @@
 F = TypeVar("F", bound=features._Feature)
 
 
-def dispatch(kernels: Dict[Any, Optional[Callable]]) -> Callable[[Callable[..., F]], Callable[..., F]]:
-    """Decorates a function to automatically dispatch to registered kernels based on the call arguments.
+class Dispatcher:
```

Comment:
Nope, this is not only refactoring. Before this PR, we had a regular decorator that defines the wrapper callables as local functions. This is fine as long as you don't need access to these functions. Since we do need this here, for example to find the supported types, I extracted the inner functionality into a class. Recall, that this was the case in #5407 until we stripped away the functionality since it already was aligned with the transforms.